services/movie.py

```diff
@@ -0,0 +1,32 @@
+from db.models import Movie
+from django.db.models import QuerySet
+from typing import List
+
+
+def get_movies(genres_ids: List[int] = None,
+               actors_ids: List[int] = None
```

Comment:
They accepts 
Optional[List[int]] because they have by default None parameter
Check in other places